src/schemas/base-types.json

```diff
@@ -37,7 +42,7 @@
 	"uint": {
 		"title": "hex encoded unsigned integer",
 		"type": "string",
-		"pattern": "^0x[1-9a-f]+[0-9a-f]*$"
+		"pattern": "^0x[0-9a-f]+[0-9a-f]*$"
```

Comment:
this now matches `0x0` fixing #106

Reply:
The problem is it also matches `0x00` which is not valid :(

Reply:
What about 0x01?

Reply:
Aight I think I got it now:
`^0x([1-9a-f]+[0-9a-f]*|0)$`

I'm assuming the left-hand-side extra zeroes after the `0x` are not valid, for example 
0x00, 0x01, 0x000001abc.
I'm assuming the correct representations should be:
0x0, 0x1, and 0x1abc.

Is that correct/better?

Reply:
Yes, exactly! Thank you.